Source/SwipeToRevealView.swift

```diff
@@ -1,8 +1,17 @@
 import UIKit
 import SnapKit
 
+@objc public protocol SwipeToRevealViewDelegate {
+  @objc optional func swipeToRevealView(view: SwipeToRevealView, didClose animated: Bool)
```

Comment:
What about little refactoring:
```swift
func swipeToRevealView(_ view: SwipeToRevealView, didCloseAnimated: Bool)
```
so it's called like this:
```swift
delegate?.swipeToRevealView(self, didCloseAnimated: true)
```
In my opinion it reads much better.